avocado-plugins-vt.spec

```diff
@@ -6,7 +6,7 @@
 
 Summary: Avocado Virt Test Plugin
 Name: avocado-plugins-vt
-Version: 41.0
+Version: %{version}
```

Comment:
This does not work (at least on my system). I tried to look for a way to read file or execute shell to get some value, but never found a way to do that. Anyway I know nothing about RPMs, so there might be a way, but I think we just have to update the VERSION and spec file manually.

Anyway the `make rpm` fails on my system so I think you should either find a way to get the version from the `VERSION` file or keep the version string here.